src/main/java/org/testobject/appium/junit/TestObjectAppiumSuite.java

```diff
@@ -159,10 +151,18 @@ public TestObjectAppiumSuite(Class<?> clazz) throws InitializationError {
 			Set<DataCenterSuite> dataCenterSuites = getDataCenterSuites();
 
 			this.perDeviceRunners = toRunners(clazz, dataCenterSuites);
+		}
+		runAllDevicesAtOnce(config);
+	}
 
-			this.setScheduler(new ThreadPoolScheduler(perDeviceRunners.size(), testObjectTimeout, config.timeoutUnit()));
+	private void runAllDevicesAtOnce(TestObject config) {
+		int numRunners = perDeviceRunners.size();
+		int timeout = Env.getTimeout()
```

Comment:
No default timeout anymore for local runs?

Reply:
The annotation `TestObject` provides the default timeout of 60 minutes.